resources/public/api/conf/1.0/other_expenses_retrieve.yaml

```diff
@@ -0,0 +1,100 @@
+summary: Retrieve Other Expenses
+description: |
+  This endpoint enables you to retrieve other expenses.
```

Comment:
Description in the spec is `This endpoint allows a developer to retrieve expenses income for trade union and patent royalties. A National Insurance number and tax year must be provided.`

Reply:
same, that's from the old spec